electrical/motor_controller.py

```diff
@@ -125,12 +126,36 @@ def motors(self, omega, vel):
         omega_l = vl * self.wheel_r
 
         # Define and cap duty cycles if they are above max
-        dc1 = omega_r / self.vm_load1
-        dc2 = omega_l / self.vm_load2
+        try: 
+            dc1 = omega_r / self.vm_load1
+            dc2 = omega_l / self.vm_load2
+        except: 
+            raise ZeroDivisionError("vm_load1 or vm_load2 is zero")
+
         if dc1 > 100:
             dc1 = 100
         if dc2 > 100:
             dc2 = 100
+        
+        if self.is_sim:
+            self.p1.ChangeDutyCycle(dc1)
+            self.p2.ChangeDutyCycle(dc2)
+        else: 
+            print("dc1: ", dc1, "and dc2: ", dc2)
```

Comment:
why are we setting dc1 and dc2 when it's a simulation? isn't dc1 and dc2 controlling avg voltage sent to the motor?

Reply:
these changes are from merging master into this branch
DC1 and DC2 are for the motor controller class and are needed -- nothing to do with the GUI simulation stuff